### PR TITLE
Clean up Stock State Helpers

### PR DESCRIPTION
**Pull request by dannyroberts**

### CODE REVIEW

File: corehq/apps/commtrack/models.py [hunk 1]
```diff
@@ -340,7 +340,93 @@ def force_wrap(cls, data):
     def wrap(cls, data):
         raise NotImplementedError()
 
-class NewStockReport(object):
+
+def xml_to_stock_report_helper(form, elem):
+    tag = elem.tag
+    tag = tag[tag.find('}')+1:] # strip out ns
```
Comment (dimagimon) [inline corehq/apps/commtrack/models.py]: E226 missing whitespace around arithmetic operator

Comment (dimagimon) [inline corehq/apps/commtrack/models.py]: E261 at least two spaces before inline comment

File: corehq/apps/commtrack/processing.py [hunk 2]
```diff
@@ -6,7 +6,7 @@
 from casexml.apps.stock.const import COMMTRACK_REPORT_XMLNS
 from corehq.apps.commtrack.exceptions import MissingProductId
 from dimagi.utils.decorators.log_exception import log_exception
-from corehq.apps.commtrack.models import CommtrackConfig, NewStockReport
+from corehq.apps.commtrack.models import CommtrackConfig, xml_to_stock_report_helper
```
Comment (dimagimon) [inline corehq/apps/commtrack/processing.py]: F401 'CommtrackConfig' imported but unused

**Comment by czue** [comment]
reviewed - looks good to me. one possible question but this is good to merge from my side if you're happy with it + testing

**Comment by dannyroberts** [comment]
The changes I made seemed logically airtight to me, but of course I can make mistakes even when being careful. I'm not sure how/what exactly to test manually, and it's hard to write unit tests because I don't have the right intuition about what each bit of code is supposed to be doing in terms of inputs and outputs. Would love your opinion/guidance here!

**Comment by czue** [comment]
my sense is that this is tested enough that it's ok. agree with the airtightness so long as you were careful about carefully searching for usages of stuff. so i think this is good to merge on my side unless you want to double check anything you did

**Comment by dannyroberts** [comment]
Yeah I've looked at all the calls to it and made sure all of the parameters they pass in are accounted for. The class is also not subclassed, and I've normalized all of the indirect calls to it so all params are explicit and none of them are passing in **kwargs.

**Comment by czue** [comment]
:+1: :tada:

**Pull request merged**